src/client.rs

```diff
@@ -982,22 +842,42 @@ impl GridView {
 // * then just call `game.run()` which runs the `Game` mainloop.
 pub fn main() {
     env_logger::init().unwrap();
-    let mut c = conf::Conf::new();
 
-    c.version       = version!().to_string();
-    c.window_width  = DEFAULT_SCREEN_WIDTH;
-    c.window_height = DEFAULT_SCREEN_HEIGHT;
-    c.window_icon   = "conwayste.ico".to_string();
-    c.window_title  = "💥 conwayste 💥".to_string();
+    let mut cb = ContextBuilder::new("conwayste", "Aaronm04Manghi")
+        .window_setup(conf::WindowSetup::default()
+                      .title(format!("{} {} {}", "💥 conwayste", version!().to_owned(),"💥").as_str())
+                      .icon("\\conwayste.ico")
+                      .resizable(false)
+                      .allow_highdpi(true)
+                      )
+        .window_mode(conf::WindowMode::default()
+                     .dimensions(config::DEFAULT_SCREEN_WIDTH as u32, config::DEFAULT_SCREEN_HEIGHT as u32)
+                     .vsync(true)
+                     );
+
+    if let Ok(manifest_dir) = env::var("CARGO_MANIFEST_DIR") {
+        let mut path = path::PathBuf::from(manifest_dir);
+        path.push("resources");
+        println!("Adding path {:?}", path);
+        cb = cb.add_resource_path(path);
+    } else {
+        println!("Not building from cargo? Okie dokie.");
+    }
 
-    // save conf to .toml file
-    let mut f = File::create("ggez.toml").unwrap(); //XXX
-    c.to_toml_file(&mut f).unwrap();
+    let ctx = &mut cb.build().unwrap();
```

Comment:
Yeah the issue lies within the Mesa driver. 

Basically there are three options since fedora is still on `17.2.4-2.fc27`:
1. Downgrade to version 17.1
2. Upgrade to [Mesa `master`](https://github.com/mesa3d/mesa)
3. [Manually patch](https://github.com/ggez/ggez/issues/194#issuecomment-353228758) version 17.2.*
4. *Secret option* See if you can get [Fedora Rawhide](https://apps.fedoraproject.org/packages/mesa)